WcaOnRails/app/controllers/admin_controller.rb

```diff
@@ -75,4 +75,12 @@ def person_data
       dob: @person.dob,
     }
   end
+
+  def compute_auxiliary_data
+  end
+
+  def do_compute_auxiliary_data
+    ComputeAuxiliaryData.perform_later unless ComputeAuxiliaryData.in_progress?
```

Comment:
Adding a condition to the job doesn't really help either though, right? Just pushes the same race a little deeper into our code.